importer/balanced/builder.go

```diff
@@ -318,3 +363,49 @@ func fillNodeRec(db *h.DagBuilderHelper, node *h.FSNodeOverDag, depth int) (fill
 
 	return filledNode, nodeFileSize, nil
 }
+
+func attachMetadataDag(db *h.DagBuilderHelper, root ipld.Node, fileSize uint64) (ipld.Node, error) {
+	// Create a 'newRoot'.
+	newRoot := db.NewFSNodeOverDag(ft.TFile)
+
+	// Add metadata DAG as first child of 'newRoot'.
+	err := addMetadataChild(db, newRoot, db.GetMetaDb().GetMetaDagRoot())
```

Comment:
So, can you confirm this: metadata (if available) is always the left most child.